cli/src/commands/new.rs

```diff
@@ -44,7 +44,7 @@ use crate::ui::Ui;
 /// the new commit. This can be avoided with `--no-edit`.
 ///
 /// Note that you can create a merge commit by specifying multiple revisions as
-/// argument. For example, `jj new main @` will create a new commit with the
+/// argument. For example, `jj new @ main` will create a new commit with the
 /// `main` bookmark and the working copy as parents.
```

Comment:
nit: Swap the order of ‘the `main` bookmark’ and ‘the working copy’ as well for consistency.